bundle/edu.gemini.spModel.core/src/main/scala/edu/gemini/spModel/core/Deflated.scala

```diff
@@ -4,25 +4,35 @@ import java.io._
 import java.util.Arrays
 import java.util.zip.{ Deflater, Inflater }
 
-/**
- * A compressed value.
- * @param data the underlying byte array
- */
-final class Deflated[A] private (val data: Array[Byte]) extends Serializable {
+import scalaz.Functor
+
+/** A value that has been serialized and compressed. */
+final class Deflated[A] private (
+  private val blob: Array[Byte], 
+  private val k: Object => A
```

Comment:
Functions are serializable but problematic, so that's a good point. I'm not using the functor so I'll get rid of it.